code/src/components/ErrorBoard.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const ErrorBoard = () => {
+  return (
+    <div className="error-board">
+      <div className="api-error"> Oops, an error occured during API call!</div>
+      <img className="sad-face" src="https://media.giphy.com/media/2hg8EPKHz7HLTXYm2i/giphy.gif" alt="Sad face"/>
+    </div>
+  )
+}
+
+export default ErrorBoard
```

Comment:
Looks neat! I would like to see this in action but I'm not sure how to do that?😬  I mean, I don't want your app to break...but still😅